packages/web-pixel-ui-extensions/src/extension-points/web-pixel-api.ts

```diff
@@ -0,0 +1,23 @@
+export type Version = 'unstable';
+
+/**
+ * The following APIs are provided to the Pixel extension point.
+ */
+export interface WebPixelApi {
+  analytics: {
+    subscribe(event: string, callback: (data: any) => void): void;
```

Comment:
Will we have strong typing on this at some point?

Reply:
Also, will there eventually be a way to unsubscribe, or do we not think these extensions need that?